brainiak/matnorm_models/noise_covs.py

```diff
@@ -0,0 +1,268 @@
+import tensorflow as tf
+import numpy as np
+import abc
+from .helpers import define_scope, xx_t
+
```

Comment:
Why do you call NoiseCov* to these matrices?
What is the Noise part in them? 
It seems more realistic to call them Covariance* or Cov*. (same for the filename).

Is there any reason to initialize most of them randomly? why not to offer an initialization value option and if it is None then use the random values?